testsuite/cluster_test/src/tx_emitter.rs

```diff
@@ -101,12 +102,55 @@ impl TxEmitter {
             thread::sleep(Duration::from_millis(10)); // Small stagger between starting threads
         }
         println!("Threads started");
+        let prometheus = Prometheus::try_new_from_environment();
+        if let Some(prometheus) = prometheus {
+            run_stat_loop(prometheus);
+        }
         for join_handle in join_handles {
             join_handle.join().unwrap();
         }
     }
 }
 
+fn run_stat_loop(prometheus: Prometheus) {
+    thread::sleep(Duration::from_secs(30)); // warm up
+    loop {
+        thread::sleep(Duration::from_secs(10));
+        if let Err(err) = print_stat(&prometheus) {
+            println!("Stat error: {:?}", err);
+        }
+    }
+}
+
+fn print_stat(prometheus: &Prometheus) -> failure::Result<()> {
+    let step = 10;
+    let end = unix_timestamp_now();
+    let start = end - Duration::from_secs(30); // avg over last 30 sec
+    let tps = prometheus.query_range(
+        "irate(consensus_gauge{peer_id='8be62727', op='last_committed_version'}[1m])".to_string(),
```

Comment:
Why do you need to specify the peer id here? Wouldn't average work here? In case one peer id is better, can we map a validator name in the cluster to the peer id of Prometheus?